src/align.cpp

```diff
@@ -35,21 +37,31 @@ void test_pcl(Registration& reg, const pcl::PointCloud<pcl::PointXYZ>::ConstPtr&
   double single = std::chrono::duration_cast<std::chrono::nanoseconds>(t2 - t1).count() / 1e6;
   std::cout << "single:" << single << "[msec] " << std::flush;
 
-  // 100 times
+  if (visualize) {
+    pcl::visualization::PCLVisualizer viewer;
+    viewer.addPointCloud(target, "tgt");
+    viewer.setPointCloudRenderingProperties(pcl::visualization::PCL_VISUALIZER_COLOR, 1, 0, 0, "tgt");
+    viewer.addPointCloud(aligned, "aligned");
+    viewer.setPointCloudRenderingProperties(pcl::visualization::PCL_VISUALIZER_COLOR, 0, 1, 1, "aligned");
+    while (!viewer.wasStopped()) viewer.spin();
+    viewer.close();
+  }
+
+  // 10 times
   t1 = std::chrono::high_resolution_clock::now();
-  for (int i = 0; i < 100; i++) {
+  for (int i = 0; i < 10; i++) {
```

Comment:
I would like to keep the iteration number to 100, because 10 is too small to benchmark GPU-based methods.